src/routes/index.js

```
@@ -7,6 +7,7 @@ const adminInvitation = require('~/routes/adminInvitation')
 const question = require('~/routes/question')
 const resourcesCategory = require('~/routes/resourcesCategory')
 const offer = require('~/routes/offer')
+const subject = require('~/models/subject')
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)